apps/web/src/app/vault/components/vault-items/vault-items.stories.ts

```diff
@@ -69,6 +70,15 @@ export default {
             },
           } as Partial<StateService>,
         },
+        {
+          provide: SettingsService,
+          useValue: {
+            disableFavicon$: new BehaviorSubject(false).asObservable(),
+          } as Partial<SettingsService>,
+          async getDisableFavicon() {
+            return false;
+          },
+        },
```

Comment:
issue: I think you missed a `}` bracket somewhere
```suggestion
        {
          provide: SettingsService,
          useValue: {
            disableFavicon$: new BehaviorSubject(false).asObservable(),
            async getDisableFavicon() {
              return false;
            },
          } as Partial<SettingsService>,
        },
```